### ISSUE TITLE
Missing AS numbers 

### ISSUE THREAD

**Issue by zelima** [cybergreen-net/refdata-asn#1]
There are AS numbers in scanned data that are not present in reference data
Currently I manually added several of them. But surely will be many others in the future.

We need to have reliable source with "all" possible ASNs as reference - **manually** adding thing is just a temporary hack and **not** solution

**Comment by zelima** [comment]
@rufuspollock This is coming back Each time the new data arrives. Need help to think for solution
Also same issue with countries https://github.com/cybergreen-net/refdata-country/issues/3 (this did note came back yet, but still)

**Comment by rufuspollock** [comment]
@zelima can you give examples.

/cc @aaronkaplan @chorsley @kxyne

**Comment by zelima** [comment]
Ones I added manually 
```
45075,,CN
19399,,SE
64322,,CO
53207,,BR
56096,,XY
63990,,XY
38717,,PH
1325556,,IN
45664,,PH
17781,,CN
65557,,SA
290012147,,US
45606,,TH
132371,,TH
64121,,US
132304,,NZ
9805,,CN
264179,,BR
12845938,,PL
4264800030,,CA
65555,,BR
131599,,TW
65538,,US
65536,,ES
4264800033,,CA
64111,,US
53056,,BR
53128,,BR
65539,,SI
4200000007,,US
55973,,CN
59199,,JP
65537,,IL
65610,,ZA
131314,,IN
4200000006,,US
64339,,CO
```

**Comment by aaronkaplan** [comment]
@zelima : for all countries, periodically we could do 
``wget https://stat.ripe.net/data/country-asns/data.json?resource=$countrycode&lod=1``

Example:
``https://stat.ripe.net/data/country-asns/data.json?resource=nl&lod=1``

Output:
```
{
"status": "ok",
"server_id": "stat-app8",
"status_code": 200,
"version": "0.1",
"cached": false,
"see_also": [],
"time": "2017-01-26T07:06:35.552594",
"messages": [
[
"info",
"Query time has been set to the latest time (2017-01-26 00:00 UTC) data is available for."
],
[
"info",
"Registy data is valid for 2017-01-24"
]
],
"data_call_status": "development",
"process_time": 4671,
"build_version": "2017.1.10.190",
"query_id": "f6a1ca22-e395-11e6-8ebf-0050568835e6",
"data": {
"resource": [
"nl"
],
"query_time": "2017-01-26T00:00:00",
"latest_time": "2017-01-26T00:00:00",
"lod": [
"1"
],
"countries": [
{
"resource": "nl",
"routed": [
"30720",
"59393",
"38915",
"60358",
"47112",
"24586",
"24587",
"28685",
"20495",
"38930",
"8211",
"48283",
"20504",
"40985",
"200730",
"12315",
"59426",
"12323",
...
```

**Comment by zelima** [comment]
@aaronkaplan Yes, I know that. And think this is done here, right? https://github.com/cybergreen-net/reference-data/blob/master/country2asn/fetch.sh
Problem is that there are ASNs besides that Eg: `64339` - came in from latest scan, but is not present there https://raw.githubusercontent.com/cybergreen-net/reference-data/master/country2asn/import.csv
So my guess is that  https://stat.ripe.net/ is not enough

Besides that, there is one more thing to consider. I'm using this script https://github.com/cybergreen-net/reference-data/blob/master/asn2asname/fetch_asn_name_db.py to get reference data for AS. Notice that this one uses different source - http://www.cidr-report.org/as2.0/autnums.html
2 reasons I'm doing this: 
- This one includes all info we need - `as-number`, `as-name`, `country`
- It contains much more AS numbers then data from https://stat.ripe.net/ (is more complete)

But even this one is missing ones I listed above.

**Comment by aaronkaplan** [comment]
> On 26 Jan 2017, at 09:11, Irakli Mchedlishvili <notifications@github.com> wrote:
> 
> @aaronkaplan Yes, I know that. And think this is done here, right? https://github.com/cybergreen-net/reference-data/blob/master/country2asn/fetch.sh
> Problem is that there are ASNs besides that Eg: 64339 - came in from latest scan, but is not present there https://raw.githubusercontent.com/cybergreen-net/reference-data/master/country2asn/import.csv
> So my guess is that https://stat.ripe.net/ is not enough
> 

or that we simply need to keep the data updated ?

> Besides that, there is one more thing to consider. I'm using this script https://github.com/cybergreen-net/reference-data/blob/master/asn2asname/fetch_asn_name_db.py to get reference data for AS. Notice there this one uses different source - http://www.cidr-report.org/as2.0/autnums.html
> 2 reasons I'm doing this:
> 
> 	• This one includes all info we need - as-number, as-name, country
> 	• It contains much more AS numbers then data from https://stat.ripe.net/
> But even this one is missing ones I listed above.
> 

We need to compare why there is a difference. After all - RIPE should be a very very authoritative source.
